swift/ql/src/queries/Security/CWE-327/BrokenCryptoAlgorithm.swift

```diff
@@ -0,0 +1,15 @@
+import CryptoKit
+import Foundation
+
+let SECRET_KEY = SymmetricKey(size: .bits256)
+
+func sendEncrypted(plaintext: Data, to channel: Channel) {
+    channel.send(cipher.encrypt(plaintext)) // BAD: weak encryption
```

Comment:
Is `cipher` placeholder?

Also is `Channel` a Swift class, third party library, or something we're just implying the software of the example has?

Reply:
It's just a placeholder, there's nothing quite equivalent in Swift's standard library, which prefers Actors for cross-thread communication, so I'll just replace this with a `return` since this query does not care about where ciphertext ends up getting used.